package.json

```diff
@@ -8,25 +8,27 @@
   "scripts": {
     "start": "nodemon src/app.js",
     "start-production": "export NODE_ENV=production && nodemon src/app.js",
-    "test": "export NODE_ENV=test && yarn validate && jest --passWithNoTests",
+    "test": "export NODE_ENV=test && jest --passWithNoTests --forceExit",
     "lint": "eslint --ignore-path .gitignore .",
     "format": "prettier --ignore-path .gitignore --write \"**/*.+(js|json)\"",
     "validate": "yarn format && yarn lint",
     "validate-staged": "lint-staged",
-    "postinstall": "husky install && husky add .husky/pre-commit \"yarn validate-staged\" && husky add .husky/pre-push \"yarn test\""
+    "postinstall": "husky install && husky add .husky/pre-commit \"yarn validate\" && husky add .husky/pre-push \"yarn test\""
```

Comment:
why did you changed these scripts?

Reply:
I deleted `yarn validate` from the test script so we don't wait for validation everytime we run the test. This way the validation will happen only upon committing.
I added `--forceExit` to the test so the terminal will exit the test after it's done instead of us typind cttrl+c everytime.
I deleted the `-staged` from the postinstall script because it's redundant and the `yarn validate` will make sure that the code is validated before being committed.